### PR TITLE
bump dependencies

### PR DESCRIPTION
**Pull request by Asone**
Bumps dependencies to the latest versions of `prost` and `tonic`.

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -79,7 +79,7 @@ impl Lnd {
             Lnd::connector(certificate_bytes).map_err(LndConnectError::Connector)?;
 
         let transport = tonic::transport::Endpoint::new(destination)
-            .and_then(move |d| d.connect_with_connector_lazy(https_connector))
+            .and_then(move |d| Ok(d.connect_with_connector_lazy(https_connector)))
```
Comment (lsunsi) [inline src/lib.rs:82]: Could this be a `map`?

Comment (Asone) [reply src/lib.rs]: Seems to work, i've committed the change as requested

**Comment by Asone** [comment]
I just saw the commit signing constraint. 
I'll configure my machine up today to rebase the commit history and sign it.

**Comment by Asone** [comment]
made a `git rebase HEAD~2` to rebase adding commit signing. 

Hope it will be good for you, otherwise feel free to give me guidance.

**Pull request merged**